pkg/scheduler/util/predicate_helper.go

```diff
@@ -91,8 +92,8 @@ func (ph *predicateHelper) PredicateNodes(task *api.TaskInfo, nodes []*api.NodeI
 		}
 	}
 
-	//workqueue.ParallelizeUntil(context.TODO(), 16, len(nodes), checkNode)
-	workqueue.ParallelizeUntil(ctx, 16, allNodes, checkNode)
```

Comment:
looks like node predict and job updater worker num are separate, but only used one parameter here,  maybe we can separate or add some comments.